src/openrct2/rct1/S4Importer.cpp

```diff
@@ -1246,7 +1246,7 @@ class S4Importer final : public IParkImporter
         dst->mini_golf_current_animation = src->mini_golf_current_animation;
         dst->mini_golf_flags = src->mini_golf_flags;
 
-        sprite_move(src->x, src->y, src->z, dst);
+        dst->MoveTo({ src->x, src->y, src->z });
         invalidate_sprite_2(dst);
```

Comment:
just a thought but we don't need to do any invalidation during an import as the whole screen will be redrawn after load.